pkg/controller/template/render.go

```diff
@@ -147,7 +147,7 @@ func platformStringFromControllerConfigSpec(ic *mcfgv1.ControllerConfigSpec) (st
 		return "", fmt.Errorf("cannot generate MachineConfigs when no platformStatus.type is set")
 	case platformBase:
 		return "", fmt.Errorf("platform _base unsupported")
-	case configv1.AWSPlatformType, configv1.AlibabaCloudPlatformType, configv1.AzurePlatformType, configv1.BareMetalPlatformType, configv1.GCPPlatformType, configv1.OpenStackPlatformType, configv1.LibvirtPlatformType, configv1.OvirtPlatformType, configv1.VSpherePlatformType, configv1.KubevirtPlatformType, configv1.PowerVSPlatformType, configv1.NonePlatformType:
+	case configv1.AWSPlatformType, configv1.AlibabaCloudPlatformType, configv1.AzurePlatformType, configv1.BareMetalPlatformType, configv1.GCPPlatformType, configv1.IBMCloudPlatformType, configv1.OpenStackPlatformType, configv1.LibvirtPlatformType, configv1.OvirtPlatformType, configv1.VSpherePlatformType, configv1.KubevirtPlatformType, configv1.PowerVSPlatformType, configv1.NonePlatformType:
```

Comment:
This change is needed when you have platform specific templates added here https://github.com/openshift/machine-config-operator/tree/master/templates/common, see this function usage at https://github.com/openshift/machine-config-operator/blob/master/pkg/controller/template/render.go#L195. I would say, since there is no ibm specific template added yet so it is working as expected.

Reply:
We can either merge it now or later on when an ibm specific template gets added. Let us know what is the preference.

Reply:
ahh, thanks for the explanation @sinnykumari . i misunderstood this function, we don't have an ibm specific template so i gather we can close this PR.